ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -268,12 +268,43 @@ - (void)getUserVideo:(NSDictionary *)constraints
   }
 
   if (videoDevice) {
-    // TODO: Actually use constraints...
-    RTCAVFoundationVideoSource *videoSource = [self.peerConnectionFactory avFoundationVideoSourceWithConstraints:[self defaultMediaStreamConstraints]];
-    // FIXME The effort above to find a videoDevice value which satisfies the
-    // specified constraints was pretty much wasted. Salvage facingMode for
-    // starters because it is kind of a common and hence important feature on
-    // a mobile device.
+    NSMutableDictionary *mandatory = [[NSMutableDictionary alloc]init];
+    [mandatory setObject:@"720" forKey:@"maxHeight"];
+    [mandatory setObject:@"1280" forKey:@"maxWidth"];
+    [mandatory setObject:@"30" forKey:@"maxFrameRate"];
+    NSDictionary *video = constraints[@"video"];
```

Comment:
we already have `id videoConstraints = constraints[@"video"];` above in https://github.com/smiletime/react-native-webrtc/blob/2836169d659e7f1fce66fce15ec9e386fae8f8ca/ios/RCTWebRTC/WebRTCModule%2BRTCMediaStream.m#L219